Add new FAQ about metapackage and include install snippets to intro (#2516)
Qiskit/documentation: docs/faq.mdx

> IBM Quantum Composer. 2023. url: https://<span>quantum.ibm.com/composer</span>

> ## What happened to Qiskit Aer, Qiskit Nature, Qiskit Experiments, etc.?

Reviewer: ```suggestion
## What happened to Python distributions such as Qiskit Aer, Qiskit Nature, and Qiskit Experiments?
```

> ## What happened to Python distributions such as Qiskit Aer, Qiskit Nature, and Qiskit Experiments?

Reviewer 2: I think we should be more explicit here and warn people that since March 2024 (or whatever date we released the metapackage change) the command `pip install qiskit` will only install the Qiskit SDK (and link to the github repo), not the other packages.

Author: I added another couple sentences at the beginning of this section. Let me know if it seems sufficient!

> Before the release of the Qiskit SDK v1.0, Qiskit was comprised of many different Python distributions such as: `qiskit-terra`, the compiler core; `qiskit-aer`, the high-performance simulator; the original IBM Quantum™ provider; and several now-obsolete packages providing particular exploratory algorithmic or experiment-running features. 

Reviewer: ```suggestion
Before the release of the Qiskit SDK v1.0, Qiskit was comprised of many different Python distributions, such as `qiskit-terra`, the compiler core; `qiskit-aer`, the high-performance simulator; the original IBM Quantum™ provider; and several now-obsolete packages providing particular exploratory algorithmic or experiment-running features. 
```

> These separate software packages were combined together into what was called the Qiskit *metapackage* which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0 and each of the individual software distributions need to be installed separately. 

Reviewer: ```suggestion
These separate software packages were combined into what was called the Qiskit *metapackage*, which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0, and now each of the individual software distributions need to be installed separately. 
```

> These separate software packages were combined into what was called the Qiskit *metapackage*, which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0, and now each of the individual software distributions need to be installed separately. 

Reviewer: ```suggestion
These separate software packages were combined into what was called the Qiskit *metapackage*, which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0, and now each of the individual software distributions needs to be installed separately. 
```

Reviewer: whoops, I missed this first time around!

> Check out the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find the package you may be looking for. You can also read the [1.0 migration guide](./migration-guides/qiskit-1.0-installation#why-these-changes-happened) for more information about why this change happened.

Reviewer: ```suggestion
Visit the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find the package to suit your needs. You can also read the [v1.0 migration guide](/migration-guides/qiskit-1.0-installation#why-these-changes-happened) for more information about this change.
```

> IBM Quantum Composer. 2023. url: https://<span>quantum.ibm.com/composer</span>

## What happened to Python distributions such as Qiskit Aer, Qiskit Nature, and Qiskit Experiments?

> Since the release of the Qiskit SDK v1.0, the command `pip install qiskit` will **only** install the [Qiskit SDK](https://github.com/Qiskit/qiskit). This means that some modules that you have used in the past, such as `qiskit-aer` or `qiskit-terra`, now have to be installed separately.

Reviewer 2: ```suggestion
Since the release of the Qiskit SDK v1.0, the command `pip install qiskit` will **only** install the [Qiskit SDK](https://github.com/Qiskit/qiskit). This means that some modules that you have used in the past, such as `qiskit-aer` or `qiskit-nature`, now have to be installed separately.
```

technically they cant install terra separately bc it doesnt exist anymore

Before the release of the Qiskit SDK v1.0, Qiskit was comprised of many different Python distributions, such as `qiskit-terra`, the compiler core; `qiskit-aer`, the high-performance simulator; the original IBM Quantum™ provider; and several now-obsolete packages providing particular exploratory algorithmic or experiment-running features. 

These separate software packages were combined into what was called the Qiskit *metapackage*, which allowed users to install all of these components together through `pip install qiskit`. This metapackage architecture was removed with the release of the Qiskit SDK v1.0, and now each of the individual software distributions needs to be installed separately. 

> Visit the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find the package to suit your needs. You can also read the [v1.0 migration guide](/migration-guides/qiskit-1.0-installation#why-these-changes-happened) for more information about this change.

Reviewer 2: ```suggestion
If you want to install the packages that used to be included in the Qiskit metapackage you can visit the [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) to find a range of packages to suit your needs. You can also read the [v1.0 migration guide](/migration-guides/qiskit-1.0-installation#why-these-changes-happened) for more information about this change.
```

The ecosystem sentence sounded a bit disjointed so added a bit more context
